haiku-apps/bescreencapture/bescreencapture-2.8.0.recipe

```diff
@@ -44,6 +33,7 @@ BUILD_PREREQUIRES="
 	makefile_engine
 	cmd:gcc
 	cmd:make
+	multimarkdown
```

Comment:
Move this line just between makefile_engine and cmd:gcc (sorting).

Reply:
Currently I have issues with git on the Haiku VM, so I can't do this.
Could you fix it this time ? Or this should wait since there's a fix

Reply:
Sure, could push some changes, if you would like to check afterwards then. :)

Reply:
Just pushed some changes, could you check if it's still ok there? (build it with ffmpeg5-tools, still runs with ffmpeg6 tools installed, so it can use eiter version).